app/pages/onboarding/06-save-key/save-key.tsx

```diff
@@ -15,15 +16,14 @@ export const SaveKey: React.FC = () => {
   return (
     <Onboarding>
       <OnboardingTitle>Save your Secret Key</OnboardingTitle>
+      <OnboardingBackButton onClick={() => history.push(routes.SECRET_KEY)} />
```

Comment:
I feel like this could be abstracted to accept a route prop instead of having to pass history each time?

Reply:
I did think about this, such as just doing history back 1. Though there are some cases this wouldn't work (view secret key again action). So not too bothered about being explicit.

Reply:
ah yeah, i meant something like this `<OnboardingBackButton to={routes.SECRET_KEY} />`, still explicit, but less verbose. either way is fine with me :)

Reply:
Got it that makes more sense. I didn't need to add `useHistory` to add this, but if I do will make this change.